scout/adapter/mongo/institute.py

```diff
@@ -65,50 +65,22 @@ def update_institute(
         if not institute_obj:
             raise IntegrityError("Institute {} does not exist in database".format(internal_id))
 
-        updates = {"$set": {}}
+        updates = {"$set": {}, "$unset": {}}
         updated_institute = institute_obj
 
         if sanger_recipient:
-            user_obj = self.user(sanger_recipient)
-            if not user_obj:
-                raise IntegrityError("user {} does not exist in database".format(sanger_recipient))
```

Comment:
I think I like this, but it is a bit of a change. Obviously nice to be able to redirect to accounts that are not "users" yourself, but on the minus side of this you dont get a check that the email is correct. If you like, you could break out a small function for Sanger updates and document the intention in a docstring. But it is smallish already, so all good in the current function docstring. 😊

Reply:
The thing is, you are forced to provide the email of an existing user if you are adding a recipient using the CLI, but if you add a recipient from the web page of the institute settings, then you can add whichever email you like (works like this on current main). I figured this check is not important at all then in the CLI

Reply:
No, agreed, the risk of misspelling should be about the same! 😁 Just saying that could be worth a couple of doc words somewhere.

Reply:
I think it is fine. I wouldn't document a feature that is not there any more. And even when it was there it wasn't used if you added the sanger user via web form

Reply:
Unless we want to add the check that the user exist also when you add it using the web form?